ankimorphs/__init__.py

```diff
@@ -51,6 +51,7 @@
 from .ankimorphs_db import AnkiMorphsDB
 from .extra_settings import ankimorphs_extra_settings, extra_settings_keys
 from .extra_settings.ankimorphs_extra_settings import AnkiMorphsExtraSettings
+from .filters import register_filters
```

Comment:
the project has multiple types of filters so the 'filters' name is too ambiguous

Reply:
I started using the name template-filters to disambiguate.